playbook/app/pb_kits/playbook/pb_contact/contact.rb

```diff
@@ -35,8 +37,12 @@ def contact_icon
       def formatted_contact_value
         if contact_type == "email"
           contact_value
+        elsif contact_type == "extension" && contact_value.match(/^\d{4}$/)
+          contact_value
+        elsif contact_type != "email" && contact_type != "extension"
+          number_to_phone(formatted_value, area_code: true, raise: true)
         else
-          number_to_phone(formatted_value, area_code: true)
+          ""
```

Comment:
This area makes me nervous. Before, we had an else that would call `number_to_phone` now we don't. Can you explain?

Reply:
You're right, reviewing it makes me realize that it'd not get to the last `else` cuz everything that's not an `email` nor `extension` will be resolved on the condition before. My idea was to return an empty string when a `contact_type` wasn't specified. My suggestion is to return to the original code as the example below, what do you think?

```
if contact_type == "email"
  contact_value
elsif contact_type == "extension" && contact_value.match(/^\d{4}$/)
  contact_value
else
  number_to_phone(formatted_value, area_code: true, raise: true)
end
```

Reply:
I think the way you have it written in your comment from 5 days ago is the safest. 
Go ahead and adjust to that, and i'll give this a final review.